editor/editor_system.cpp

```diff
@@ -170,6 +172,18 @@ bool EditorSystem::initialize(const JSON::json& systemData)
 		colors[ImGuiCol_ModalWindowDimBg] = ImVec4(1.00f, 0.98f, 0.95f, 0.73f);
 	}
 
+	// Load json theme data
+	for (const auto& entry : std::filesystem::directory_iterator("editor/themes"))
```

Comment:
Register this into types.h too. We avoid writing down `std::*` around the engine. Typedef it to `DirectoryIterator` most likely.